Rakefile

```diff
@@ -2,13 +2,15 @@
 # encoding: utf-8
 
 require 'bundler'
-require 'bundler/gem_tasks'
+require 'bundler/gem_helper'
 require 'rake/testtask'
 require 'passgen'
 require 'train'
 require_relative 'tasks/maintainers'
 require_relative 'tasks/spdx'
 
+Bundler::GemHelper.install_tasks name: 'inspec'
```

Comment:
Would it make sense to have a Rake task for building the inspec-core gem, as well?